### PR TITLE
Test spipe with bps-limited echo server

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

**Comment by gperciva** [comment]
This arose from your comment https://github.com/Tarsnap/spiped/pull/207#issuecomment-539735409.  While working on it, I realized that this particular test made more sense on its own.  The "pushbits-only" test will make a reappearance after this PR is in.

File: tests/08-send-data-spipe-limited-bps.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Goal of this test:
+# - ensure that spipe can close the sending side of the TCP
+#   connection, while the receiving side keeps working.
+
+### Constants
+echo_limited_output="${s_basename}-echo-limited-output.txt"
```
Comment (cperciva) [inline tests/08-send-data-spipe-limited-bps.sh]: Is this used anywhere?

File: tests/nc-server/main.c [hunk 2]
```diff
@@ -11,11 +14,59 @@
...
+	struct timespec ts;
+
+	ts.tv_sec = msec / 1000;
+	ts.tv_nsec = (msec % 1000) * 1000000;
+	nanosleep(&ts, NULL);
+
+	return (0);
+}
+
+/* Send a message, limited to bits per second.  Send in bursts of 10ms. */
```
Comment (cperciva) [inline tests/nc-server/main.c]: bits, or bytes?

**Comment by cperciva** [comment]
REBASE commits look good.

**Comment by gperciva** [comment]
Rebased, ready for merge.

**Pull request merged**